ComputerArchitecture/Ywoosang/2025-08-22-Ywoosang-CPU_메모리_구조_Overview.md

```
@@ -0,0 +1,327 @@
+* 수 GB\~수십 GB 수준으로 사용됨
+
+### 4. 보조 저장장치 (SSD, HDD)
+
+* 비휘발성 저장소, 전원이 꺼져도 데이터 유지
+* CPU 입장에서는 가장 느린 계층
+
+## CPU 캐시 일관성 문제
+
+여러 CPU 코어가 같은 메모리 주소를 각자 캐시에 저장할 때 값이 달라질 수 있음. 이를 캐시 일관성(Cache Coherency) 문제라고 함
+
+* 메모리에 변수 A = 5
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)